src/Web/Constants.cs

```diff
@@ -1,9 +1,11 @@
 ﻿namespace Microsoft.eShopWeb.Web;
 
+
 public static class Constants
 {
     public const string BASKET_COOKIENAME = "eShop";
     public const int ITEMS_PER_PAGE = 10;
     public const string DEFAULT_USERNAME = "Guest";
     public const string BASKET_ID = "BasketId";
+    public const string BASKET_NAME = "eShopnonWebApp";
```

Comment:
Is that correct.? I feel the value has to be "eShopOnWebApp". Also, are we using this constant anywhere else in the codebase.?

Reply:
yeah good question, it isn't used at all.